source/computer/commands.rst

```diff
@@ -63,54 +62,556 @@ Linux/macOS 下有成百上千个命令，每个命令都有众多选项。这
 
     -   其他
 
+        - `df`_
+        - `du`_
         - `ssh`_
         - `scp`_
         - `rsync`_
+        - `wget`_
+        - `top`_
 
 ----
 
 cat
 ---
 
+``cat`` 命令名来源于 con\ **cat**\ enate（即“拼接”）。该命令可以输出某个文件的
+内容，或将多个文件\ **拼接**\ 起来。
+
+将一个文件的内容打印到标准输出::
```

Comment:
not accurate, but easier to understand 
```suggestion
将一个文件的内容输出到终端::
```

Reply:
Should we use:

```
# 将一个文件的内容输出到终端
$ cat file

# 将文件 file1 和 file2 合并到一个文件 target_file 中
$ cat file1 file2 > target_file
```

I think it can save a lot of space.

Reply:
Yes, I think this is a good way.

Reply:
done in f954636.